README.md

```diff
@@ -36,94 +35,28 @@ Here's a list of benefits you will get if you use this project:
-- `main.py` contains the code to respond to webhook events.
-
-## Set Up
-
-**NOTE**: Currently, this app is only used internally. See [milestone](https://github.com/getsentry/sentry-github-actions-app/milestone/1) for required work to support ingesting data from other orgs.
-
-These are the steps you will need to set up this backend and Github app for your organization.
-
-The steps of the next two sections are related. You will be generating keys and variables that are need to configure each component (the backend and the Github App), thus, you will need to go back and forth.
-
-### Sentry SaaS (or self-hosted Sentry)
+- Create a Sentry project to gather your Github CI data
+  - You don't need to select a platform in the wizard or any alerts
+  - Find the DSN key under the settings of the project you created
+- Create a public repo name `.sentry`
+- Create a file in that repo called `sentry_config.ini` with these contents (adjust your DSN value)
 
-In Sentry.io:
-
-- Create a project to track errors of the backend itself (`APP_DSN` in section below)
-- Create a project to track Github jobs (`SENTRY_GITHUB_DSN` in section below)
-
-### Backend deployment
-
-- Deploy the app to a service (e.g. GCP) and make it publicly reachable
-  - Take note of the app's URL and use it when creating the Github app webhook
-
-Common environment variables:
-
-- `APP_DSN`: Report your deployment errors to Sentry
-- `GH_WEBHOOK_SECRET`: Secret shared between Github's webhook and your app
-  - Create a secret with `python3 -c 'import secrets; print(secrets.token_urlsafe(20))'` on your command line
-- `SENTRY_GITHUB_DSN`: Where to report your Github job transactions
-- `LOGGING_LEVEL` (optional): To set the verbosity of Python's logging (defaults to INFO)
-
-Github App specific variables:
-
-- `GH_APP_ID` - Github App ID
-  - When you create the Github App, you will see the value listed in the page
-- `GH_APP_INSTALLATION_ID` - Github App Installation ID
-  - Once you install the app under your Github org, you will see it listed as one of your organizations' integrations
-  - If you load the page, you will see the ID as part of the URL
-- `GH_APP_PRIVATE_KEY` - Private key
-  - When you load the Github App page, at the bottom of the page under "Private Keys" select "Generate a private key"
-  - A .pem file will be downloaded locally.
-  - Convert it into a single line value by using base64 (`base64 -i path_to_pem_file`)
-
-For local development, you need to make the App's webhook point to your ngrok set up. You should create a new private key (a .pem file that gets automatically downloaded when generated) for your local development and do not forget to delete the private key when you are done.
-
-### The Github App
-
-After creating the Github App for the first time there are some changes you need to make:
+```ini
+[sentry-github-actions-app]
+; DSN value of the Sentry project you created in the
+dsn = https://foo@bar.ingest.sentry.io/foobar
```

Comment:
As per conversation with Markus, it is probably fine to have a DSN publicly available.

Alternatively, we could use GH secrets but I don't know how comfortable customers would be to grant secrets permission.